README.md

```diff
@@ -17,19 +17,30 @@ This section captures projects that are capable of issuing SPIFFE identities to
 * Kong
 * Dapr
 * Athenz
+* [Teleport](https://github.com/gravitational/teleport)
 
 ## SPIFFE-aware Software
 This section captures projects that are capable of using SVIDs to identify themselves and their peers (i.e. they support SPIFFE authentication). If a piece of software is listed in the `SPIFFE Issuers` section, then it is implied that it is SPIFFE-aware.
 
 * Envoy
 * SPIRE
 * gRPC
-* Ghostunnel
+* [Ghostunnel](https://github.com/ghostunnel/ghostunnel) - Light-weight server or client TLS proxy with support for basic access control.
```

Comment:
Having a link to Ghostunnel but no other tool triggers my symmetry and consistency urge :).

While we are touching this page, we can very well add links to other tools too.

@amoore877 — is there a convention/reason against this (i.e., adding links to only a subset of the items but not all of them)?

I mean I don't know the URL to Knox, and I very much would like to click on a link that I can bounce from here to there instead of searching the web and possibly clicking to a unrelated competitor's website.

Reply:
Honestly - it was mostly that Ghostunnel struck me as something that folks new to SPIFFE might not know about but coudl find very useful. I'd argue that every item on the list should have a link and an explanation of why it is relevant ( just as is done on most "awesome" lists )

Reply:
> While we are touching this page, we can very well add links to other tools too.

I wouldn't block this PR on wanting to add more things :) another PR can expand

agreed we should just link to everything plus description